mapboxgl/utils.py

```diff
@@ -10,6 +10,8 @@ def df_to_geojson(df, properties=None, lat='lat', lon='lon', precision=None):
 
     if not properties:
         properties = list(df.columns)
+        properties.remove(lat)
```

Comment:
@ustroetz let's add properties `iff` they are not the identified latitude or longitude columns from the dataframe (as opposed to adding them, then removing them).

Reply:
Make sense. I'll change that.

Reply:
👍